_includes/footer.html

```diff
@@ -4,9 +4,15 @@
 		{% if page.lang == "de-DE" %}
 		Unsere Seite in anderen Sprachen&nbsp;:
 		<a href="/en-US/">English</a>
+		<a href="/es-ES/">Spanish</a>
```

Comment:
Though it might be a moot point, we may want to have a locale-specific name for the language instead of the English name of it, i.e. change this to `Espan~ol` (sorry can't type the tilde).